pkg/drivers/vfkit/vfkit.go

```diff
@@ -313,8 +341,10 @@ func isBootpdError(err error) bool {
 }
 
 func (d *Driver) Stop() error {
-	if err := d.SetVFKitState("HardStop"); err != nil {
-		return err
+	if err := d.SetVFKitState("Stop"); err != nil {
+		// vfkit may be already stopped, shutting down, or not listening.
```

Comment:
I would still like to see we try to Force Stop using internal command first 
and if Vfkit force stop is not doing the right thing we should create an issue
and comment the link to the issue here
so when that issue is fixed we can clean up this code

Reply:
I'll create an issue for vfkit and add a link here.

I can add fallback using "HardStop", and then fallback to sending a signal. But the HardStop is very likely to fail so this will only add more failure logs to the process without improving anything. I can post another pr with this change later for testing.

Reply:
We have this issue in vfkit:
https://github.com/crc-org/vfkit/issues/277